python-sdk/docs/development/RELEASE.md

```diff
@@ -2,7 +2,15 @@
 
 ## Updating the changelog
 
-The first step to creating a release is identifying what changed from the previous release and adding a description to the [CHANGELOG.md](../CHANGELOG.md)
+The first step to creating a release is identifying what changed from the previous release and adding a description to the [CHANGELOG.md](../CHANGELOG.md).
+
+One way to identify what changed from the previous release is to use a comparison URL between the previous release branch and the main branch.
+
+Example -
+[https://github.com/astronomer/astro-sdk/compare/1.3.3...main](https://github.com/astronomer/astro-sdk/compare/1.3.3...main)
```

Comment:
is this also show the commit which has been released in a minor release? let's say I'm looking for a changelog of 1.5.0 and looking at the result of URL https://github.com/astronomer/astro-sdk/compare/1.4.0...main

Reply:
@pankajastro  It will show you the diff that is, only the new commits added to the main branch since you released 1.4.0.

For example:

commit - 1  <- main branch
commit - 2
commit - 3   <- 1.4.0 release tag
commit - 4
commit - 5

```https://github.com/astronomer/astro-sdk/compare/1.4.0...main```
The above link will result in:

commit - 1 <- main branch 
commit - 2